src/DataGrid.php

```diff
@@ -1491,7 +1491,7 @@ public function filterSucceeded(NetteForm $form): void
 			return;
 		}
 
-		$values = (array) $form->getValues();
+		$values = $form->getUnsafeValues('array');
```

Comment:
Attention!

BC break in version 6.9.3 according this PR

```
		$inlineAdd->onSubmit[] = function (ArrayHash $values): void {
			bdump($values);
			$this->flashMessage('Record was added!', 'success');
			$this->redrawControl('flashes');
		};
```

Previously typeint was `Nette\Utils\ArrayHash` and typehint is `array` now.